CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
   - `POST <legacy-http-prefix>/push`
   - `POST /push`
   - `POST /ingester/push`
+* [CHANGE] Chunks GCS object storage client uses the `fields` selector to limit the payload size. #3218
```

Comment:
It's not a change impacting the user. Also, it's not even a change for embedders. It's more an `[ENHANCEMENT]`. We should also mention it affects only listing objects in the bucket.

```suggestion
* [ENHANCEMENT] Chunks GCS object storage client uses the `fields` selector to limit the payload size when listing objects in the bucket. #3218
```